client/blocks/product-purchase-features/product-purchase-features-list/video-audio-posts.jsx

```diff
@@ -13,15 +13,21 @@ import { localize } from 'i18n-calypso';
 import PurchaseDetail from 'components/purchase-detail';
 import paths from 'lib/paths';
 
-export default localize( ( { selectedSite, translate } ) => {
+export default localize( ( { selectedSite, plan, translate } ) => {
+	const storageLimit =
+		plan === 'business'
+			? 'The Business plan also adds unlimited file storage.'
+			: 'The Premium plan also adds 10GB of file storage.';
```

Comment:
Don't forget about i18n. Just run it through `translate`.

Reply:
Actually, we can't use string concatenation with variables like this. There are a few solutions to this (see [i18n-calypso](https://github.com/Automattic/i18n-calypso#translate-method) for all options of `translate`) but it depends on whether we want to have the ability to translate the whole feature description at once or just the particular sentences.

If whole at once, just duplicate the message and change the storage text according to `plan`. If just the storage sentence, we could translate the first part of the description into one variable, then the second storage part into another variable and then concatenate them in the `description` field.

I would say putting the whole description into `translate` is preferred, even though we have to duplicate parts of the text.